modules/apache-airflow/templates/values.yaml

```diff
@@ -636,7 +638,9 @@ workers:
   extraVolumeMounts: []
 
   # Select certain nodes for airflow worker pods.
-  nodeSelector: {}
+  nodeSelector: {
+    spotinst.io/node-lifecycle: "od"
+  }
```

Comment:
This document describes where this label is coming from:
https://docs.spot.io/ocean/features/labels-and-taints?id=spotinstionode-lifecycle